### PR TITLE
built out admin panel

### PR DESCRIPTION
**Pull request by itsWill**
I rule and have completed some work on Case Manager that's ready for review!

Built out the admin panel according to https://github.com/colinxfleming/dcaf_case_management/issues/686.

Things todo:
- [ ] clean up inline css
- [ ] extract partials
- [ ] formate the table data correctly

But we can do these once we have more functionality implemented

screen shot 2016-10-20 at 5 54 34 pm

@colinxfleming

### CODE REVIEW

**Comment by mebates** [comment]
Hey guys - this didn't get finished from a design perspective and what you built isn't what we decided on for design. Please touch base with @nimiro !

**Comment by itsWill** [comment]
@mebates I just picked the first page, I understand the frontend design is not ready yet so this mainly sets up the backend, we can then change the view according to the design you and @nimiro come up with once its decided.

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,8 +1,13 @@
 # Additional user methods in parallel with Devise -- all pertaining to call list
 class UsersController < ApplicationController
   before_action :retrieve_patients, only: [:add_patient, :remove_patient]
+  before_action :admin_user, only: [:new, :index]
```
Comment (colinxfleming) [inline app/controllers/users_controller.rb]: Thanks for abstracting this! Love it

File: test/controllers/users_controller_test.rb [hunk 2]
```diff
@@ -20,6 +20,22 @@ class UsersControllerTest < ActionController::TestCase
     end
   end
 
+  describe 'index method' do
```
Comment (colinxfleming) [inline test/controllers/users_controller_test.rb]: 💯 💯 💯 💯 💯  for tests!

**Comment by colinxfleming** [comment]
@mebates like @itsWill noted, this rigs up the backend so that we can throw the actual designs in. Fully expect that we'll have another pull request or three before we're 100% good to go, but this PR (setting up routing and tests) means that the next person to take a stab will be able to hit the ground running when team ux has reached an endpoint in #626 . $$$$

**Comment by mebates** [comment]
Great!

**Pull request merged**